generators/javascript/display.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * @license
+ * Visual Blocks Language
+ *
+ * Copyright 2012 Google Inc.
+ * Copyright 2020 Brady L. Hurlburt.
+ * https://developers.google.com/blockly/
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Generating JavaScript for Rovercode display blocks.
+ * @author brady.hurlburt@rovercode.com (Brady L. Hurlburt)
+ */
+'use strict';
+
+goog.provide('Blockly.JavaScript.display');
+
+goog.require('Blockly.JavaScript');
+
+Blockly.JavaScript['display_message'] = function(block) {
+  var value_message = Blockly.JavaScript.valueToCode(block, 'MESSAGE', Blockly.JavaScript.ORDER_ATOMIC);
+  var code = 'displayMessage(String('+ value_message +'));\n';
```

Comment:
This seems to work fine on `String("foo")`, `String(1234)`, and `String(true)`.

Reply:
What about a variable?

Reply:
Added that to the rovercode-all.xml. Seems to work well.

Reply:
`displayMessage(String(a));`